src/common/ast-path.js

```diff
@@ -16,7 +16,8 @@ class AstPath {
 
   /** @type {object} */
   get node() {
-    return this.stack.at(-1);
+    // eslint-disable-next-line unicorn/prefer-at
```

Comment:
```suggestion
    // eslint-disable-next-line unicorn/prefer-at -- Extremely slow on Node.js v16 and v18
```

So we'll know when to remove it.